lametro/templates/lametro/about.html

```diff
@@ -196,7 +196,13 @@ <h3>Credits</h3>
 
         <h3>Contact us</h3>
 
-        <p>Have a question about information you were seeking to find, or explanations of how Metro works? Send us an email <a href="mailto:boardreport@metro.net">boardreport@metro.net</a>.</p>
+        <div class="col-sm-12 no-pad-mobile">
+            <p>For questions about or comments for the Metro Board, please contact <a href="mailto:boardclerk@metro.net">boardclerk@metro.net</a>. Written public comment can be mailed to: Board Administration, 1 Gateway Plaza, Mail Stop 99-3-1, Los Angeles, CA 90012.</p>
```

Comment:
Weird, this content is indented relative to the header here. Possible to have it flush with the header, like the Credits section above?

![Screen Shot 2022-01-11 at 11 20 05 AM](https://user-images.githubusercontent.com/12176173/148990712-5146e9ac-ef0a-413a-8959-7bcd46f68a2b.png)